src/video/win32_v.cpp

```diff
@@ -346,7 +346,7 @@ static LRESULT HandleIMEComposition(HWND hwnd, WPARAM wParam, LPARAM lParam)
 		if ((lParam & GCS_COMPSTR) && DrawIMECompositionString()) {
 			/* Read composition string from the IME. */
 			LONG len = ImmGetCompositionString(hIMC, GCS_COMPSTR, nullptr, 0); // Length is always in bytes, even in UNICODE build.
-			wchar_t *str = (wchar_t *)_alloca(len + sizeof(wchar_t));
+			std::wstring str(len, '\0');
 			len = ImmGetCompositionString(hIMC, GCS_COMPSTR, str, len);
```

Comment:
```suggestion
			len = ImmGetCompositionString(hIMC, GCS_COMPSTR, str.data(), len);
```
and missing `.c_str()` on lines 355 and 360 (comments not available for theses, so can't make suggestion).